frontend/src/pages/modelRegistrySettings/useRoleBindings.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { listRoleBindings } from '~/api';
+import { RoleBindingKind } from '~/k8sTypes';
+import useFetchState, { FetchState } from '~/utilities/useFetchState';
+
+const useRoleBindings = (): FetchState<RoleBindingKind[]> => {
```

Comment:
Can we name this something that reflects its filter? `useModelRegistryRoleBindings` maybe?